src/Quadmath.jl

```diff
@@ -82,39 +82,20 @@ end
 # and passed on the xmm registers, matching the x86_64 ABI for __float128.
 const Cfloat128 = NTuple{2,VecElement{Float64}}
 
-struct Float128 <: AbstractFloat
-    data::Cfloat128
-    function Float128(data::Cfloat128)
-        new(data)
-    end
-end
+primitive type Float128 <: AbstractFloat 128 end
```

Comment:
This seems like a breaking change?

Reply:
Yeah, maybe bump the version number?

Reply:
Yeah. If we're going to make a breaking release, might as well take the opportunity to do 1.0.0 now.

Reply:
Is this actually breaking? I would be very surprised if anyone was using the layout of this type. That said, no objection to making a 1.0 tag...